home.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="home.css">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"/>
+</head>
+<body>
+    <header>
+            <a class="main_page" href="/"><img class="logo" src="logo.png" alt="판다마켓 로고"></a>
```

Comment:
❗️ 수정요청
해당 a 태그를 클릭해보시면 해당하는 페이지가 없다는 화면이 나옵니다. 앵커 태그의 href를 /로 설정해두셨지만 지금 레포에서는 해당하는 파일이 없기 때문입니다.
이럴때 요구 사항대로 구현하기 위해서는 home.html이 아니라 index.html로 파일명을 변경하시면 잘 작동할거에요~

```suggestion
            <a class="main_page" href="/"><img class="logo" src="logo.png" alt="판다마켓 로고"></a>
```